### DISCUSSION TITLE
How do you run the project locally after cloning it?

### DISCUSSION THREAD

**Discussion by 2div** [StreamController/StreamController#257] [General]
Hey there,  

I just cloned the repo to my machine. Can you guide me on how to run the project locally after cloning it? Also, where can I find the style file for the "Get Started" button in the navbar? I'm a React developer, not a Python one, and I'm eager to contribute to this repo.  

Thanks!

**Comment by Core447** [comment]
You can find the installation/run instructions [here](https://streamcontroller.github.io/docs/latest/installation/#github). All the css stylings are in `style.css`
